packages/asset/contracts/Asset.sol

```diff
@@ -78,6 +80,7 @@ contract Asset is
         __ERC1155Supply_init();
         __ERC2771Handler_init(forwarder);
         __ERC1155Burnable_init();
+        __Ownable_init();
```

Comment:
We need to add functions in the deploy scripts somewhere to transfer the ownership of the contracts to the admin

Reply:
I think that is better to take an address as argument and then call _transferOwnership instead of depending on the deploy scripts.